<launch>

    <!-- depth_lib -->
    <node pkg="depth_lib" type="depth.py" name="depth_lib">
        <remap from="rgb_image" to="/hsrb/head_rgbd_sensor/rgb/image_rect_color"/>
        <remap from="depth_image" to="/hsrb/head_rgbd_sensor/depth_registered/image_raw"/>
        <remap from="camera_info" to="/hsrb/head_rgbd_sensor/depth_registered/camera_info"/>
    </node>

    <!-- yolo_tf -->
    <node pkg="yolo_tf" type="yolo.py" name="yolo_test" args="--ckpt /workspace/src/wrs2020/yolo_tf/data/yolov3.ckpt --names /workspace/src/wrs2020/yolo_tf/data/coco.names --ckpt1 /workspace/src/wrs2020/yolo_tf/data/ycb_200326/data.ckpt-449000 --names1 /workspace/src/wrs2020/yolo_tf/data/ycb_200326/data.names --type1 classifier" output="screen">
        <remap from="image" to="/hsrb/head_rgbd_sensor/rgb/image_rect_color"/>
    </node>
   
</launch>
